R/utils.R

```diff
@@ -83,6 +83,22 @@ git_init <- function(path = ".") {
   )
 }
 
+#' Provide a singular interface for hook calls to docopt
+#'
+#' docopt provides different processing for a single string
+#' than an array/vector, and so this function wraps docopt
+#' to
+#'
+#' @param doc \code{character} vector with command line specification
```

Comment:
We use markdown syntax with {roxygen2}, so please just use backticks around _character_. Same for the rest of the function documentation.